socorro/lib/versionutil.py

```diff
@@ -2,88 +2,80 @@
 # License, v. 2.0. If a copy of the MPL was not distributed with this
 # file, You can obtain one at http://mozilla.org/MPL/2.0/.
 
+import semver
+
 
 class VersionParseError(Exception):
     """Raised if the version isn't parseable"""
 
-    pass
-
-
-def validate_version(version):
-    """Validate a version."""
-    if not (version and isinstance(version, str)):
-        return False
-
-    version = version.split(".")
-
-    # Versions should have at least two parts: X.Y
-    if len(version) < 2:
-        return False
 
-    # First part should consist of one or more ascii digits
-    if not (len(version[0]) > 0 and version[0].isdigit()):
-        return False
+def generate_semver(version):
+    """Convert a version to semver.
 
-    return True
```

Comment:
This is crap validation and we "get it for free" by try to parse the version. So I nixed this.